pkg/interactor/data.go

```diff
@@ -0,0 +1,110 @@
+package interactor
+
+import (
+	"log"
+	"strconv"
+	"time"
+
+	. "github.com/Igoraamc/knot-cloud-storage/pkg/entities"
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type DataDAO struct {
+	Server   string
+	Database string
+}
+
+var db *mgo.Database
+
+const (
+	COLLECTION = "things"
+)
+
+func (m *DataDAO) Connect() {
+	session, err := mgo.Dial(m.Server)
+	if err != nil {
+		log.Fatal(err)
+	}
+	db = session.DB(m.Database)
+}
```

Comment:
What do you think about returning an `error` here, to better handle/debug these cases? :grimacing: